src/styles/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 166px;
```

Comment:
The card width is set to 166px, which may not meet the task requirements. Ensure that the width is set according to the specifications provided in the task description.